### ISSUE TITLE
Surface Marker Does Not Show In Original Clean

### ISSUE THREAD

**Issue by libbybapa** [BEEmod/BEE2-items#1568]
It would be great if it made the square display in some way differently.  The checkerboard effect that is shown in clean would be fine.  It is a useful item and it would be nice to be able to use it in original clean.

**Comment by TeamSpen210** [comment]
It's somewhat intentional since that's the point of Original Clean - not changing surface textures. It always uses the same texture set that angled panels and flip panels also use.

**Comment by libbybapa** [comment]
I understand that all the surfaces are the same in Original Clean, but is there a reason to not allow the surface marker to work?  As an example, here is a screenshot from the map I am currently working on.  As you can see the floor is white in order to accommodate flings.  The center square is important as it is directly under the button that activates the cube dropper.  Without some distinguishing characteristic to the center square, it's an unnecessary pain to accurately shoot the portal there.  As such, I'm relegated to using a static flip panel which is unnecessarily heavy on the item count.  A surface marker would be preferable.  Is there a downside to allowing its use in Original Clean?  

![20170209161539_1](https://cloud.githubusercontent.com/assets/19520567/22807304/766d7ae6-eee3-11e6-9463-1d8efde90e6e.jpg)
![20170209161554_1](https://cloud.githubusercontent.com/assets/19520567/22807311/7e47d4e6-eee3-11e6-8bec-83a1ff6748d9.jpg)

**Comment by Byzarru** [comment]
What is the downside to using BEE Clean?

**Comment by libbybapa** [comment]
The downside to BEE clean is that it randomizes the textures and gives a different feel.  More to the point, overlapping partial blocks can show a texture clash.

I'm not particularly partial to the checkerboard although it's fine.  Would something else be more suitable to the style and still allow demarking a particular square and still allowing it to be portalable, etc.

**Comment by TeamSpen210** [comment]
Your walls have randomised textures.

If I made it checkerboard / H-pattern, it'd also apply to flip and angled panels.

**Comment by libbybapa** [comment]
In Original Clean the wall textures are not randomized.  The do go to a smaller grid with partial squares, though.  In the screenshot, I have some angled blocks on the sides, so the pattern goes to a smaller grid.

**Comment by ZombiePwner1** [comment]
The whole point of original clean is to not have the wall textures randomized. Original clean is mean't to be the original puzzlemaker style.

**Comment by TeamSpen210** [comment]
Maybe I'll add the `special` textures (that's what I call them in the configs) to Original Clean, but disable them on angled panels.

**Comment by ZombiePwner1** [comment]
@TeamSpen210 Make an option to either have the special textures enabled or not.

**Comment by libbybapa** [comment]
Yes, ZombiePwner1.  I totally agree.  In some maps I like that the textures are not randomized.  I have still run into the desire to be able to demark specific important panels.

If the surface marker worked in Original Clean, it would be my preference for the checkerboard *not* to show up on angled and flip panels.

**Comment by libbybapa** [comment]
...or making it optional on angled panels or flips.  Options are always good, IMO.

**Comment by libbybapa** [comment]
How about panel sized Aperture Logo that still allows the panel to be portalable?  That would be cool...  :)

**Comment by TeamSpen210** [comment]
The checkerboards are used in canon for this purpose
